service/src/main/java/greencity/service/ubs/UBSManagementServiceImpl.java

```diff
@@ -1379,7 +1379,13 @@ private Payment buildPaymentEntity(Order order, ManualPaymentRequestDto paymentR
             .orElseThrow(() -> new EntityNotFoundException(EMPLOYEE_NOT_FOUND));
         eventService.save(OrderHistory.ADD_PAYMENT_MANUALLY + paymentRequestDto.getPaymentId(),
             employee.getFirstName() + "  " + employee.getLastName(), order);
-        eventService.save(OrderHistory.ORDER_PAID, OrderHistory.SYSTEM, order);
```

Comment:
I see we call 
```
 eventService.save(OrderHistory.ADD_PAYMENT_MANUALLY + paymentRequestDto.getPaymentId(),
            employee.getFirstName() + "  " + employee.getLastName(), order);
```
And after that, we check if `(order.getOrderPaymentStatus() != null)` and if it is not `null` we call 
```
    eventService.save(OrderHistory.ORDER_PAID, OrderHistory.SYSTEM, order);
```
or
```
eventService.save(OrderHistory.ORDER_HALF_PAID, OrderHistory.SYSTEM, order);
```
Based on this logic we could have a situation when payment is `ADD_PAYMENT_MANUALLY` but not `ORDER_PAID` or `ORDER_HALF_PAID`
Is this case valid?

Reply:
I'll ask viktoriya about it, I think that we may need to verify also for unpaid orders, but I'll check

Reply:
There are four Payment Statuses: Paid, Half Paid, Unpaid, Payment Refunded.
For the payment entity, we don't need to have cover for all events, because those other 2 events are covered in other methods, like when you cancel an order. In case we don't pay o half pay the order, I don't think that it should display an event for that